### DISCUSSION TITLE
oracledb is returning "None" for a specific field

### DISCUSSION THREAD

**Discussion by arjan-strijland** [oracle/python-oracledb#281] [Q&A]
I'm a bit baffled at this one.
I am trying to fetch some very simple information from the Oracle database. This is a stripped down example to show the issue:
```python
for row in cursor.execute("SELECT PRB_NUMBER, IMPL_BASELINE FROM CORP.VW_EUR_PRB_PROBLEM WHERE PRB_NUMBER = '11693'"):
   raise Exception(row)
```

This gives me:
```python
Exception: (11693, None)
```

If I execute the same query via a powershell using sqlplus:
```bash
Connected to:
Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
Version 19.20.0.0.0

SQL> SELECT PRB_NUMBER, IMPL_BASELINE FROM CORP.VW_EUR_PRB_PROBLEM WHERE PRB_NUMBER = '11693';

PRB_NUMBER IMPL_BASELINE
---------- ------------------------
     11693 24/NVCS-B4patch
```

Anyone has any idea why the IMPL_BASELINE field is returned as "None" in Python?
The field is a VARCHAR2(24) field if that helps.

Any advice is much appreciated!

**Comment by anthony-tuininga** [comment]
Unfortunately, that doesn't really help much! There are a lot of possibilities so more information will be needed on the query that is being executed, the definition of the view and if any PL/SQL is being executed to render the contents of that view, etc. Can you provide those?

**Comment by arjan-strijland** [comment]
My bad, turns out I was accidentally comparing 2 different instances of the same database that were not synchronized.
So the result I was seeing was correct.
